app/models/referee_interface/reference_safeguarding_form.rb

```diff
@@ -0,0 +1,47 @@
+module RefereeInterface
+  class ReferenceSafeguardingForm
+    include ActiveModel::Model
+    attr_accessor :any_safeguarding_concerns, :safeguarding_concerns, :candidate
+
+    validate :presentce_of_any_safeguarding_concerns
```

Comment:
Isn't this just:

```ruby
validate :any_safeguarding_concerns, presence: true
```

Reply:
There may be an easier way to do this, but I need a custom error message with the candidate name. 

I also tried:
`validate_presence_of :any_safeguarding_concerns, message: "Some error message"`

But it didn't like string interpolation